src/App.scss

```diff
@@ -89,4 +89,24 @@ p {
     margin-top: 0px;
     width: 100%;
     overflow: hidden;
+}
+
+
+/*
+    `footer-to-bottom` wraps the body and footer for each page.
+    It spaces the two containers out so that the footer is pushed 
+    to the bottom of the view if the body contents do not fill up 
+    the whole view. Otherwise, the footer will rest below the 
+    body content.
+*/
```

Comment:
:D nice description

Reply:
I learned from a frand that comments are good :-)